common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/IDToken.java

```diff
@@ -187,6 +187,11 @@
      */
     public static final String UPDATED_AT = "updated_at";
 
+    /**
+     * The authority that issued the token
+     */
+    public static final String ISSUER = "iss";
```

Comment:
where is this used ?

Reply:
I was initially using this in MSAL PR: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/843

But removed it from there, and hard coded the `iss` string to remove that PR's dependency on this common PR as the tests failed in this PR.

Reply:
Since that MSAL PR no longer depends on this PR, I'll remove this constant for now.